packages/slice-machine/components/ChangesSectionHeader/index.ts

```diff
@@ -0,0 +1 @@
+export { ChangesSectionHeader } from "./ChangesSectionHeader";
```

Comment:
I guess more will be added to this file later

Reply:
it's just so that you don't have to import the component from `ChangesSectionHeader/ChangesSectionHeader`
otherwise I can call the actual `ChangesSectionHeader.tsx` file `index.tsx`, but I find that less clear cuz all our files become being called index ahah

Can change it if you prefer otherwise tho

Reply:
makes sense i guess :)